src/rules/experimental.d.ts

```diff
@@ -0,0 +1,39 @@
+export type Severity = 0 | 1 | 2;
```

Comment:
I assume this was a file to test your setup?
If this is the case, please remove this from source.
Have a look into the vitest suite that also do type-check tests.